GOVERNANCE.md

```diff
@@ -1,36 +1,87 @@
-whithin the framework established by the guidelines published in the `haskell-cryptography/governance` repository.
-The referent person is  autonomous but held accountable. This doesn't prevent a decision proces that is involving multiple parties in a "flat"
-governance structure for a project, but the referent is the one person that is in charge of making these decisions official, answering for them and enforcing them.
+This role involves enacting the goals of the Organisation, and being in charge of the management of the project,
+whithin the framework established by the guidelines published in the [haskell-cryptography/governance](https://github.com/haskell-cryptography/governance) repository.
+The referent person is  autonomous but held accountable. This doesn't prevent a decision process that involves multiple parties in a "flat"
+governance structure for a project, but the key person is the one person that is in charge of making these decisions official,
+answering for them and enforcing them.
 
 ### Services related to the code and users
 
 As a support structure for open-source maintainers, `haskell-cryptography` has the ability to host projects of the Haskell ecosystem related to cryptography,
-should their maintainer decide that this is the best course of action for their project. By transferring a project to the organisation, the authors of a project
-do not implictly relinquish their permissions, authorship and ownership of the project, and benefit from the collaborators of the organisation to help them
-with tasks they need help with. 
+should their maintainer decide that this is the best course of action for their project.
+By transferring a project to the organisation, the authors of a project do not implictly relinquish their permissions, authorship and ownership
+of the project, and benefit from the collaborators of the organisation to help them with tasks they need help with.
 
-Projects can be donated for historical preservation or to seek new leadership. 
+Projects can be donated for ongoing maintenance and improvement, or to seek new leadership. 
+The organisation intends to allow package maintainers to maintain control of their packages, if they wish to, while making it easier to bring together effort to improve the state of cryptography in the Haskell ecosystem.
+In the interest of keeping a healthy and secure ecosystem, the Core Team will have to perform some tasks on projects that were transferred to the Organisation.
+These tasks may take priority over the specific ideas of the original authors of the projects.
+We acknowledge that the authors have insights that we lack and know their domains better than the Core Team may, but in the interest of the ecosystem the needs of our contributors may outweigh the ideals of the original authors. This is a flexible policy and consensus with the authors will always be preferred.
```

Comment:
Perhaps also add something about the Core Team having access to all the projects so they can make urgent changes if the key person isn't responsive. If we had a Log4J style bug that sat sound for a few weeks because no one could make the changes that would be problematic. 

I think it would be wise to put something in here about requiring that the core team does have commit rights and can release any project in the organisation, but that those rights are supposed to only be used in urgent circumstances, and having those rights should eb a requirement for adding the project to the organisation.

Reply:
Yup', got it